concepticondata/conceptlists/DellaRosa-2010-417.tsv

```diff
@@ -0,0 +1,418 @@
+DellaRosa-2010-417-134	134	Morale	moral			181	267	526	494	274	542	645
+DellaRosa-2010-417-135	135	Motivo	motive			223	235	559	370	350	547	456
+DellaRosa-2010-417-136	136	Noia	boredom			274	403	632	327	553	494	363
+DellaRosa-2010-417-137	137	Numero	number	1356	NUMBER	459	620	652	226	575	375	332
+DellaRosa-2010-417-138	138	Odio	detestation			224	326	480	370	383	470	521
+DellaRosa-2010-417-139	139	Onore	honour			164	241	429	488	395	615	488
+DellaRosa-2010-417-140	140	Originalità	originality			288	268	523	415	368	568	548
+DellaRosa-2010-417-141	141	Orrore	horror			305	356	491	400	475	450	363
+DellaRosa-2010-417-142	142	Ottimismo	optimism			244	258	559	460	385	529	463
+DellaRosa-2010-417-143	143	Ozio	idleness			268	384	445	450	494	480	568
+DellaRosa-2010-417-144	144	Pace	peace	1338	PEACE	221	308	567	303	469	465	537
+DellaRosa-2010-417-145	145	Panico	panic	1000	ANXIETY	288	388	507	450	404	400	512
+DellaRosa-2010-417-146	146	Paradiso	paradise			171	412	560	305	483	605	647
+DellaRosa-2010-417-147	147	Passione	passion			312	452	547	525	576	474	481
+DellaRosa-2010-417-148	148	Paura	scare	781	FEAR (FRIGHT)	338	379	550	190	515	447	308
+DellaRosa-2010-417-149	149	Pazienza	patience			319	279	568	330	476	476	420
+DellaRosa-2010-417-150	150	Peccato	pity	1141	PITY	188	268	440	390	358	490	600
+DellaRosa-2010-417-151	151	Pericolo	danger	1810	DANGER	323	361	511	231	506	394	397
+DellaRosa-2010-417-152	152	Piacere	pleasure	3916	PLEASURE	342	412	605	309	491	469	350
+DellaRosa-2010-417-153	153	Preferenza	preference			300	274	473	325	321	506	492
+DellaRosa-2010-417-154	154	Preghiera	prayer			375	379	495	270	550	418	268
+DellaRosa-2010-417-155	155	Preparazione	preparation			284	267	559	366	400	483	515
+DellaRosa-2010-417-156	156	Pretesto	pretence			212	187	363	544	200	565	628
+DellaRosa-2010-417-157	157	Principio	principle			159	215	467	485	279	560	618
+DellaRosa-2010-417-158	158	Protesta	protest			358	435	521	388	494	394	375
+DellaRosa-2010-417-159	159	Protezione	protection			287	373	521	316	409	425	465
+DellaRosa-2010-417-160	160	Punizione	punishment	390	PUNISHMENT	436	394	524	235	540	368	269
+DellaRosa-2010-417-161	161	Ragione	reason	2968	REASON	245	212	585	396	390	547	480
+DellaRosa-2010-417-162	162	Reazione	reaction			291	309	509	422	425	421	431
+DellaRosa-2010-417-163	163	Reputazione	reputation			215	213	411	569	282	594	619
+DellaRosa-2010-417-164	164	Resistenza	resistance			291	306	474	443	430	482	469
+DellaRosa-2010-417-165	165	Responsabilità	responsibility	3471	OBLIGATION (DUTY)	262	232	576	418	438	550	611
+DellaRosa-2010-417-166	166	Riduzione	abatement			268	196	330	453	263	500	626
+DellaRosa-2010-417-167	167	Rimedio	remedy	1372	MEDICINE	284	241	516	367	353	463	488
+DellaRosa-2010-417-168	168	Risentimento	resentment			219	227	394	494	296	496	515
+DellaRosa-2010-417-169	169	Risultato	result	3969	RESULT	377	306	600	342	415	394	381
+DellaRosa-2010-417-170	170	Saggezza	wisdom			185	224	500	475	392	510	591
```

Comment:
Another good candidate for a new mapping, and not super rating-list-specific.

Reply:
@MiraAhmedovic Could you open an issue linking this PR where we can decide whether we want to include WISDOM as a new concept?